cognite/client/data_classes/events.py

```diff
@@ -96,6 +96,7 @@ class EventFilter(CogniteFilter):
         asset_external_ids (List[str]): Asset External IDs of equipment that this event relates to.
         root_asset_ids (List[Dict[str, Any]]): This parameter is deprecated. Use assetSubtreeIds instead. Only include events that have a related asset in a tree rooted at any of these root assetIds.
         asset_subtree_ids (List[Dict[str, Any]]): Only include events that have a related asset in a subtree rooted at any of these assetIds (including the roots given). If the total size of the given subtrees exceeds 100,000 assets, an error will be returned.
+        data_set_id (Union[Dict[str, Any], EndTimeIsNull]): No description.
```

Comment:
can we rename EndTimeIsNull ?

Reply:
poor generator, not quite sure how it ended up with that choice of types, spec is more sane. Will try to fix